### PR TITLE
Add distinguish classes to links and comments

### PR DESCRIPTION
**Pull request by 13steinj**
Adds "distinguished" and "distinguished-(type)" classes to links and comments.

As requested [here](https://www.reddit.com/r/ideasfortheadmins/comments/48pq2i/adding_a_distinguished_or_similar_class_to_links/)

### CODE REVIEW

File: r2/r2/templates/comment_skeleton.html [hunk 1]
```diff
@@ -41,7 +41,14 @@
...
 
 <%def name="thing_css_class(what)">
-  ${parent.thing_css_class(what)} ${"stickied" if getattr(thing, 'is_sticky', None) else ""} ${"collapsed" if thing.collapsed else "noncollapsed"} ${"collapsed-for-reason" if hasattr(thing, "collapsed_reason") else ""}
+  ${parent.thing_css_class(what)} ${"stickied" if getattr(thing, 'is_sticky', None) else ""} ${"collapsed" if thing.collapsed else "noncollapsed"} ${"collapsed-for-reason" if hasattr(thing, "collapsed_reason") else ""} ${distinguished_css_class(what)}
+</%def>
+
+<%def name="distinguished_css_class(what)">
+  <% distinguish_type = getattr(what, 'distinguished', None) %>
+  %if distinguish_type and distinguish_type not in ('gold', 'gold-auto'):
+    ${'distinguished distingished-%s' % distinguish_type}
```
Comment (13steinj) [inline r2/r2/templates/comment_skeleton.html]: MFW I can't spell: :(

**Comment by novovnz** [comment]
What's the point of `.distinguished-(type)` when you could do `.distinguished.link`, and `.distinguished.comment` ?

**Comment by 13steinj** [comment]
@novovnz By `.distinguished-(type)` I mean `.distinguished-moderator`, `.distinguished-admin`, and `.distinguished-special`

**Comment by geo1088** [comment]
I'm the guy that requested this, still waiting on it to go through. This is a pretty big thing for a subreddit design I'm doing, without this it's really hacky - nigh impossible - to get this behavior. 

For context, I'm basically trying to separate mod posts from AutoMod's normal stickied by giving mod posts a different color; the only way I can do this now is by using an obscure attribute to test for the poster and make sure they're not AutoMod.

Anything?

**Comment by 13steinj** [comment]
@geo1088 the chance of getting a response is less than getting bitten by a shark and struck by lightning at the same time, I'd say. PRs are, unfortunately, very very slow for the admins to process.

**Comment by geo1088** [comment]
Yeah, I noticed. Sucks though, I guess I'll just have to stick with hardcoding into the final version :/ just thought I'd bump the thread just in case

**Comment by bsimpson63** [comment]
Sorry, but this repository is no longer accepting pull requests.

**Pull request closed without merging**